activities/transcribe.go

```diff
@@ -50,3 +51,42 @@ func Transcribe(
 		TXTPath:      paths.MustParse(filepath.Join(jobData.OutputPath, fileName+".txt")),
 	}, nil
 }
+
+type MergeTranscriptJSONParams struct {
+	MergeInput      common.MergeInput
+	DestinationPath paths.Path
+}
+
+type MergeTranscriptResult struct {
+	Path paths.Path
+}
+
+// MergeTranscriptJSON is the activity that merges a transcript JSON
+// Note that currently Norwegian is hardcoded
+func MergeTranscriptJSON(
+	ctx context.Context,
+	input MergeTranscriptJSONParams,
+) (*MergeTranscriptResult, error) {
+	log := activity.GetLogger(ctx)
+	activity.RecordHeartbeat(ctx, "MergeTranscriptJSON")
+	log.Info("Starting MergeTranscriptJSON")
+
+	targetFile := input.DestinationPath.Append("merged_transcription.no.json")
+	mergedTranscription := transcribe.MergeTranscripts(input.MergeInput)
+
+	marshalled, err := json.Marshal(mergedTranscription)
+	if err != nil {
+		return nil, err
+	}
+
+	WriteFile(ctx, WriteFileInput{
+		Path: targetFile,
+		Data: marshalled,
+	})
```

Comment:
This also returns an error if it fails. Might be smart to return it here if it occurs.